etc/node_ws/app.js

```diff
@@ -0,0 +1,98 @@
+// Defaults; configurable through environment
+const HOST = process.env.HOST || '127.0.0.1';
+const PORT = process.env.PORT || 8080;
+const SUB_PATH = process.env.SUB_PATH || '/subscribe/'
+const CLUSTER_ID = process.env.CLUSTER_ID || 'usgs'
+const CHANNEL = process.env.CHANNEL || 'anss.pdl.realtime'
+const PING_INTERVAL = process.env.PING_INTERVAL || 30000;
+const LOG_LEVEL = process.env.LOG_LEVEL || 'info';
+
+var package = require('./package.json')
```

Comment:
So far, it looks like node doesn't support imports by default; I have to enable them. I've done this, but imports still aren't working. Working on a solution.

Reply:
Yeah - Each package that I use has to have a type of "module" if I want to use ES6 imports with node. I can _instead_ use typescript; unsure whether the webservice would still run without significant changes. Otherwise, it looks like we're stuck with requires. Let me know what your preference is.

Reply:
nbd, use require for now.  All requires should use `const`

Reply:
done